dbms/src/Interpreters/Join.h

```diff
@@ -326,8 +326,6 @@ class Join
     BlocksList blocks;
     /// mutex to protect concurrent insert to blocks
     std::mutex blocks_lock;
-    /// keep original block for concurrent build
-    Blocks original_blocks;
```

Comment:
It will soonly be used when join support spill, I think we can keep it.

Reply:
Why not use `BlocksList blocks`?

Reply:
Because the block in `blocks` is changed during `insertFromBlock`, the key columns are removed or moved to the start of the block.

Reply:
Got it. Maybe we can change this behavior in the future. If so, we can also delete the copy of the right join key columns.